source/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Downloads
+---
+# Downloads
+
+The master sources are maintained in our [git repository](gitrepo.html),
+which is accessible over the network and cloned on GitHub, at
+<https://github.com/openssl/openssl>. Bugs and pull patches (issues and
+pull requests) should be filed on the GitHub repo. Please familiarize
+yourself with the [license](license.html).
+
+The table below lists the latest releases for every branch. (For an explanation
+of the numbering, see our [release strategy](/policies/releasestrat.html).)
+All releases can be found at [/source/old](old). A list of mirror sites can be
+found [here](mirror.html).
+
+*Note:* The latest stable version is the 3.0 series supported until 7th
+September 2026. This is also a Long Term Support (LTS) version. The
+previous LTS version (the 1.1.1 series) is also available and is
+supported until 11th September 2023. All older versions (including
+1.1.0, 1.0.2, 1.0.0 and 0.9.8) are now out of support and should not be
+used. Users of these older versions are encouraged to upgrade to 3.0 as
+soon as possible. Extended support for 1.0.2 to gain access to security
+fixes for that version is [available](/support/contracts.html).
+
+OpenSSL 3.0 is the latest major version of OpenSSL. The OpenSSL FIPS
+Object Module (FOM) 3.0 is an integrated part of the OpenSSL 3.0
+download. You do not need to download the 3.0 FOM separately. Refer to
+the installation instructions inside the download, and use the
+"enable-fips" compile time configuration option to build it.
+
+For an overview of some of the key concepts in OpenSSL 3.0 see the
+libcrypto [manual
+page](https://www.openssl.org/docs/man3.0/man7/crypto.html). Information
+and notes about migrating existing applications to OpenSSL 3.0 are
+available in the [OpenSSL 3.0 Migration
+Guide](https://www.openssl.org/docs/man3.0/man7/migration_guide.html)
+
+<table>
+  <tr>
+    <td>KBytes&nbsp;</td>
+    <td>Date&nbsp;&nbsp;</td>
+    <td>File&nbsp;</td>
+  </tr>
+  <!--#include virtual="index.inc" -->
+</table>
+
```

Comment:
Somehow the table is merged with the next paragraph in the https://beta.openssl.org/source/ which looks weird.